internal/api/v1beta1/app_types.go

```diff
@@ -90,17 +92,17 @@ func (v DeploymentVersion) String() string {
 }
 
 type AppDeploymentSpec struct {
-	Image           string            `json:"image"`
-	Version         DeploymentVersion `json:"version"`
-	Processes       []ProcessSpec     `json:"processes,omitempty"`
-	KetchYaml       *KetchYamlData    `json:"ketchYaml,omitempty"`
-	Labels          []Label           `json:"labels,omitempty"`
-	RoutingSettings RoutingSettings   `json:"routingSettings,omitempty"`
-	ExposedPorts    []ExposedPort     `json:"exposedPorts,omitempty"`
-
 	// ImagePullSecrets contains a list of secrets to pull the image of this deployment.
 	// If this list is defined, app.Spec.DockerRegistrySpec is not used.
 	ImagePullSecrets []v1.LocalObjectReference `json:"imagePullSecrets,omitempty"`
+	Image            string                    `json:"image"`
+	Version          DeploymentVersion         `json:"version"`
+	Processes        []ProcessSpec             `json:"processes,omitempty"`
+	Resources        v1.ResourceRequirements   `json:"resources,omitempty"`
```

Comment:
I think we are good with having resources in ProcessSpec only.
Is this field used?

Reply:
Good catch, I think I accidentally left this in when I was rebasing. I'll remove it real quick.